google/cloud/bigtable/internal/default_row_reader.cc

```diff
@@ -74,6 +75,8 @@ bool DefaultRowReader::NextChunk() {
     if (absl::holds_alternative<Status>(v)) {
       last_status_ = absl::get<Status>(std::move(v));
       response_ = {};
+      retry_context_.PostCall(*context_);
+      context_ = nullptr;
```

Comment:
Maybe use an absl::optional instead of nullptr?

Reply:
I am not following. I don't use `nullptr` to signal anything the way we might with `optional::has_value()`. I just thought it would be nice to delete the `grpc::ClientContext` object when I was done with it.